src/databricks/labs/ucx/contexts/application.py

```diff
@@ -287,15 +287,15 @@ def aws_acl(self):
     @cached_property
     def principal_locations(self):
         eligible_locations = {}
-        if self.is_azure:
-            try:
+        try:
```

Comment:
Minimize `try-except` to only try code for  which we accept the error

Reply:
@nfx asked the opposite